__test__/index.test.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-env jest */
+
+import whenAllSettled from '../src'
+
+describe('whenAllSettled', () => {
+    test('should call console.error if no rejection handler is provided', async () => {
+        const promises = []
+        promises.push(Promise.reject(new Error('Failed Promise')))
```

Comment:
For readability: there seems no reason to use `.push`. I'd put the promises directly in the const definition.

Otherwise, looks all good to me. I suggest we merge it now and we can add more test(s) later.